screenA/idr0037-screenA-bulk.yml

```diff
@@ -1,4 +1,15 @@
 ---
 target: "Screen:name:idr0037-vigilante-hipsci/screenA"
-include: "../../bulk.yml"
 path: "idr0037-screenA-plates.tsv"
+continue: "true"
+transfer: "ln_s"
+# Do not exclude if we are reimporting raw data from idr0034
+# exclude: "clientpath"
+l: readers.txt
+checksum_algorithm: "File-Size-64"
+logprefix: "logs/"
+output: "yaml"
+parallel_upload: "50"
```

Comment:
Just wondering what effect a parallel upload of 50 has when there's only one `fileset` thread.

Reply:
`parallel_upload: 50` just setups 50 threads that can take uploads. If there's only one fileset or if `paralel_fileset==1`, then at most the number of files in the first fileset can be uploaded at once.

Reply:
👍  Thanks, that makes sense then.